src/electionguard_cli/e2e_steps/input_retrieval_step.py

```diff
@@ -32,9 +32,9 @@ def get_first_value(text: Optional[InternationalizedText]) -> str:
         self.print_value("Contests", len(manifest.contests))
         self.print_value("Ballot Styles", len(manifest.ballot_styles))
 
-    def get_manifest(self) -> Manifest:
+    def get_manifest(self, manifest_file: TextIOWrapper) -> Manifest:
         self.print_header("Retrieving manifest")
-        manifest: Manifest = ElectionFactory().get_simple_manifest_from_file()
+        manifest: Manifest = from_file_wrapper(Manifest, manifest_file)
```

Comment:
I considered putting this into ElectionFactory, but it seemed like it was just a passthrough and not adding much value